CHANGELOG.md

```diff
@@ -1,15 +1,24 @@
 # Change Log
+
```

Comment:
```suggestion
```

The http://keepachangelog.com actually has specifically no new line and this file follows it (as it states).

Reply:
Sure, I will drop those lines, my formatter added them.